pypeit/core/meta.py

```diff
@@ -139,6 +139,7 @@ def define_additional_meta(nlamps=20):
                        'mode': dict(dtype=str, comment='Observing mode'),
                        'object': dict(dtype=str, comment='Alternative object name (cf. target)'),
                        'obstime': dict(dtype=str, comment='Observation time'),
+                       'oscansec': dict(dtype=str, comment='Overscan section (windowing)'),
```

Comment:
I added 'oscansec' to meta, sometimes the overscan region is written in the header, it might make it easier to read.